react/features/recording/components/LiveStream/StartLiveStreamDialog.web.js

```diff
@@ -283,10 +295,8 @@ class StartLiveStreamDialog
         }
 
         if (this.state.errorType !== undefined) {
-            googleContent = (
-                <GoogleSignInButton
-                    onClick = { this._onRequestGoogleSignIn }
-                    text = { t('liveStreaming.signIn') } />
+            googleContent = ( // eslint-disable-line no-extra-parens
```

Comment:
Well, yes and no. Normally no, but in this particular case if I remove the parenthesis, it will become a multi-line component, due to the semicolon, so I have to re-add the parenthesis, then it all starts over :D

Anyhow, I reformatted it and now it doesn't need it, but then we don't utilise the full width, that Lyubo doesn't like. But I think we're good this way